ncm-opennebula/src/main/pan/quattor/aii/opennebula/default.pan

```diff
@@ -0,0 +1,49 @@
+unique template quattor/aii/opennebula/default;
+
+include 'quattor/aii/opennebula/schema';
+include 'quattor/aii/opennebula/functions';
+
+# do not overwrite/remove templates by default
+variable OPENNEBULA_AII_FORCE ?= null; 
+variable OPENNEBULA_AII_ONHOLD ?= null;
+variable OPENNEBULA_AII_FORCE_REMOVE ?= false;
+
+variable MAC_PREFIX ?= '02:00';
+
+"/system/aii/hooks/configure/" = append(dict(
```

Comment:
while cleaning up, maybe we should remove the trailing `/` in the path?
@jrha is that something for panlint to check?